azimuth/modules/base_classes/indexable_module.py

```diff
@@ -158,20 +165,34 @@ def run_postprocessing(self, output: PostProcessingIO, **kwargs) -> PostProcessi
                 f" and pipeline definitions (got {self.config.pipelines})."
             )
         postprocessors = self.config.pipelines[self.mod_options.pipeline_index].postprocessors
+        postprocessing_steps = []
         if postprocessors is not None:
             for post in postprocessors:
                 # Q: Why do we reload the postprocessors all the time?
                 # A: We should memoize it based on the module options, but that can get complicated.
                 #    It is less burdensome to just reload it.
                 fn = load_custom_object(post, **kwargs)
                 output = fn(output)
-        return output
+                # When updating to python 3.9, use .removeprefix()
+                class_name = (
+                    post.class_name[32:]
+                    if post.class_name.startswith("azimuth.utils.ml.postprocessing.")
+                    else post.class_name
+                )
```

Comment:
Two small suggestion to make this bit more robust:
1. We can use `removeprefix()`. That avoids hardcoding the length `32`.
2. Instead of hardcoding the string `"azimuth.utils.ml.postprocessing"`, which might change, we can use `PostprocessingStep.__module__`, or add `from azimuth.utils.ml import postprocessing` and use `postprocessing.__name__`.
```suggestion
                class_name = post.class_name.removeprefix(f"{PostprocessingStep.__module__}.")
```

Reply:
By the way, does that mean I should expect the `class_name` of custom steps to contain some module names (`something.something.something.`) or not? I'm asking with in mind how I treat the names in the front end.

Reply:
It all depends on what people put as a value there in their pipeline, but I don't think that should be the case by default.